### PR TITLE
W-11312482-backup-restore-revisions

### PR DESCRIPTION
**Pull request by IsaacEldridge**
@kmendiratta-crm @rohit-verma7 Can one of you take a look at this revision I did with help from Sparsh. It's basically to clarify what gets backed up vs what doesn't. Thanks!

### CODE REVIEW

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 1]
```diff
@@ -19,9 +19,35 @@ Common scenarios for using the back up and restore process include:
...
-The Runtime Fabric back up and restore process affects only the Runtime Fabric cluster, and it does not involve the control plane. Any changes you make after you perform a backup will not be present if you restore the cluster from that backup. For example, if you perform a backup, delete an application, and then restore from that backup, the application you deleted will be recreated in the cluster, but it will be marked as deleted in Runtime Manager. As a result the application's status will show as *applying* in Runtime Manager.
+== What Gets Backed Up
+
+The Runtime Fabric back up process affects only the Runtime Fabric cluster. The back up process does not back up data from the control plane.
+
+The following is backed up:
+
+* Runtime Fabric cluster details, such as controller and worker node configurations
+* Cluster-level secure properties 
+* All data in the RTF and application namespace, such as data about deployments, service accounts, svc, secrets, ingress resources, rolebindings, cluster role configurations, and ConfigMap settings
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/manage-backup-restore.adoc]: @SparshAgarwal  what is SVC?

Comment (SparshAgarwal) [reply modules/ROOT/pages/manage-backup-restore.adoc]: services

**Review by rohit-verma7 — APPROVED** [review]
LGTM

**Review by JennyHajee — APPROVED** [review]

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 2]
```diff
@@ -19,9 +19,34 @@ Common scenarios for using the back up and restore process include:
...
+
+== What Does Not Get Backed Up
+
+* Data about application state
+* Data from Ops Center
+* Data from the control plane
+
+=== Changes Made After Performing a Backup Are Not Restored
+
+Any changes you make after you perform a backup will not be present if you restore the cluster from that backup. For example, if you perform a backup, delete an application, and then restore from that backup, the application you deleted will be recreated in the cluster, but it will be marked as deleted in Runtime Manager. As a result the application's status will show as *applying* in Runtime Manager.
```
Comment (JennyHajee) [inline modules/ROOT/pages/manage-backup-restore.adoc:49]: suggest:
the application you deleted is recreated in the cluster, but it is marked as deleted in Runtime Manager. As a result, the application's status shows as *applying* in Runtime Manager.

**Review by JennyHajee — APPROVED** [review]

**Pull request merged**